### DISCUSSION TITLE
My Blocks Indexed is stuck at 72% for a long time and the docker logs show error that failed to fetch internal transactions for blocks。

### DISCUSSION THREAD

**Discussion by qianyiwangmeng** [blockscout/blockscout#9151] [Q&A] [answered]
My Blocks Indexed is stuck at 72% for a long time and the docker logs show error that failed to fetch internal transactions for blocks(block_number:582035). Is the error makes it stuck? How can I make my Blocks Indexed 100% ? help...

**Comment by qianyiwangmeng** [comment]
this is the error log 
``` "severity":"error",
    "message":"failed to fetch internal transactions for blocks: ** (ErlangError) Erlang error: [%{code: -32000, data: %{block_number: 582035, transaction_hash: \"0x7d325b648405c5a61d54859dbc86e6e7f85f4ab3165aa0cd6fa078a7634a2c35\", transaction_index: 0}, message: \"failed to get trace\"}]",
    "metadata":{
        "count":20,
        "error_count":20,
        "fetcher":"internal_transaction"
    }

**Comment by GimluCom** [comment, marked as answer]
Please try to disable the fetcher using `INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER=true` or ensure that your RPC node supports `debug_traceTransaction` / `trace_transaction` method and has it enabled.

**Reply by qianyiwangmeng** [reply]
thanks. but now I get another  quetions.
